hazelcast/src/main/java/com/hazelcast/config/MerkleTreeConfig.java

```diff
@@ -230,6 +230,16 @@ public int hashCode() {
         return result;
     }
 
+    @Override
+    public MerkleTreeConfig setName(String name) {
```

Comment:
This isn't going to be nice when/if we add merkle tree support for cache as well. The class will then be similar to `EventJournalConfig`, having `mapName` and `cacheName`.

Reply:
Then let me revert this file and I can use similar method that I have used with `EvenJournalConfig` to set name.